packages/opentelemetry-api-metrics/src/NoopMeter.ts

```diff
@@ -159,23 +159,23 @@ export class NoopMetric<T> implements UnboundMetric<T> {
 export class NoopCounterMetric
   extends NoopMetric<BoundCounter>
   implements Counter {
-  add(value: number, labels: Labels) {
+  add(value: number, labels: Labels): void {
     this.bind(labels).add(value);
   }
 }
 
 export class NoopValueRecorderMetric
   extends NoopMetric<BoundValueRecorder>
   implements ValueRecorder {
-  record(value: number, labels: Labels) {
+  record(value: number, labels: Labels): void {
     this.bind(labels).record(value);
   }
 }
 
 export class NoopBaseObserverMetric
   extends NoopMetric<BoundBaseObserver>
   implements BaseObserver {
-  observation() {
+  observation(): { observer: BaseObserver; value: number; } {
```

Comment:
Seems like there should be an `Obeservation` type here

Reply:
Fixed